recipes/usockets/all/conanfile.py

```diff
@@ -14,8 +14,10 @@ class UsocketsConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
     options = {"fPIC": [True, False],
                "with_ssl": [False, "openssl", "wolfssl"],
-               "with_libuv": [True, False]}
-    default_options = {"fPIC": True, "with_ssl": False, "with_libuv": True}
+               "eventloop": ["default", "libuv", "gcd", "asio"]}
```

Comment:
Don't add a `default` option value.
Use one in `default_options`.

It is always possible to override the default in `config_options`:
e.g.
```python
default_config_options = {"eventloop": "asio"}
def config_options(self):
    if self.settings.os == "Windows":
        self.options.eventloop = "libuv"
```

Reply:
I agree with you.
I modify eventloop's default value to "syscall" which is used on Linux(epoll), MacOSX(kqueue).
On Windows, default value is overrided  by "libuv".